app/src/organisms/ProtocolSetupParameters/ChooseCsvFile.tsx

```diff
@@ -99,23 +106,28 @@ export function ChooseCsvFile({
             </LegacyStyledText>
             <Flex css={LIST_CONTAINER_STYLE}>
               {csvFilesOnUSB.length !== 0 ? (
-                csvFilesOnUSB.map(csv => (
-                  <>
-                    {csv.length !== 0 && last(csv.split('/')) !== undefined ? (
-                      <RadioButton
-                        key={last(csv.split('/'))}
-                        data-testid={`${last(csv.split('/'))}`}
-                        buttonLabel={last(csv.split('/')) ?? 'default'}
-                        buttonValue={csv}
-                        onChange={() => {
-                          // ToDO this will be implemented AUTH-521
-                          // handleOnChange(option.value)
-                          setCsvFileInfo(csv)
-                        }}
-                      />
-                    ) : null}
-                  </>
-                ))
+                csvFilesOnUSB.map(csv => {
```

Comment:
sorry, this is my fault.
```suggestion
                csvFilesOnUSB.map(csvFilePath => {
```